app/controllers/studios_controller.rb

```diff
@@ -0,0 +1,5 @@
+class StudiosController < ApplicationController
+  def index
+    @studios = Studio.includes(:movies).all
```

Comment:
I'm curious why you included the `.includes` method here, as well as the `.all`...?